src/client/utils/url.js

```diff
@@ -4,25 +4,30 @@ import * as destLocation from './destination-location';
 import * as urlResolver from './url-resolver';
 import settings from '../settings';
 
-const HASH_RE = /#[\S\s]*$/;
+const HASH_RE                          = /#[\S\s]*$/;
+const SUPPORTED_WEB_SOCKET_PROTOCOL_RE = /^wss?:/i;
```

Comment:
Can we have a single regexp for isSupportedProtocol function (src/utils/url.js)?

Reply:
We don't support the web socket url for something other than the WebSocket class(ancor, link, iframe, etc). The server side of proxy should perform connection only for WebSocket class.

Reply:
OK